metricbeat/module/zookeeper/connections/_meta/fields.yml

```diff
@@ -0,0 +1,25 @@
+- name: connections
+  type: group
+  description: >
+    connections
+  fields:
+    - name: ip
+      type: keyword
+      description: >
+        Connection IP
+    - name: port
+      type: long
+      description: >
+        Connection Port
```

Comment:
Should this `ip` and `port` use ECS fields?

Reply:
Moved to `client.ip` and `client.port`